tools/flang1/flang1exe/semfunc.c

```diff
@@ -2727,6 +2731,67 @@ replace_formal_triples(int ast, int dscptr, int nactuals)
   }
 }
 
+/** \brief Replace the intrinsic call of LBOUND/UBOUND on assumed-shape formal.
+ *
```

Comment:
Please add a line here to explain why we only need to handle `A_BINOP`, `A_CONV`, etc. in the `switch` statement in this function. Our internal version has this:

```c
/** \brief Replace the intrinsic call of LBOUND/UBOUND on assumed-shape formal.
 * Model after replace_formal_triples which replaces triples in bounds.
 *
```

Reply:
The Minor nit has been fixed.